src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -6093,6 +6093,13 @@ checkNullable:
             End If
         End Sub
 
+        Friend Shared Function CheckFeatures(feature As Feature, opts As VisualBasicParseOptions) As Boolean
```

Comment:
It looks to that the helper in the Scanner already handles experimental features correctly. There is no reason to add an additional helper. Also, see my other comment at the call-site for this new helper.